PubMatic/src/main/java/com/applovin/mediation/adapters/PubMaticMediationAdapter.java

```diff
@@ -246,6 +266,19 @@ public void loadAdViewAd(final MaxAdapterResponseParameters parameters, final Ma
         adView.pauseAutoRefresh();
     }
 
+    @SuppressLint("MissingPermission")
+    @Override
+    public void loadNativeAd(final MaxAdapterResponseParameters parameters, @Nullable final Activity activity, final MaxNativeAdAdapterListener maxNativeAdAdapterListener)
+    {
+        final String bidResponse = parameters.getBidResponse();
+
+        log("Loading native ad");
```

Comment:
NIT: Please update the spacing around the log message to keep the consistency.
```suggestion
        log( "Loading native ad" );
```